qcodes/instrument/parameter.py

```diff
@@ -1364,6 +1364,13 @@ def get(self):
             else:
                 return state['value']
 
+    def get_timestamp(self):
```

Comment:
Add type annotation?
```suggestion
    def get_timestamp(self) -> datetime:
```

Reply:
Agreed 👍